modules.md

```diff
@@ -123,12 +123,12 @@ This will import all public names such as `say_hi` but would not import `__versi
 
 ## The `dir` function {#dir-function}
 
-Built-in `dir()` function returns list of names defined by an object.
+The built-in `dir()` function returns the list of names defined by an object.
 If the object is a module, this list includes functions, classes and variables, defined inside that module.
 
 This function can accept arguments.
-If the argument is the name of the module, function returns list of names from that specified module.
-If there is no argument, function returns list of names from the current module.
+If the argument is the name of a module, the function returns the list of names from that specified module.
+If there is no argument, it returns the list of names from the current module.
```

Comment:
Would prefer "the function" instead of "it" - repetition is okay when it aids clarity.